rfcs/0000-policy_target_scope.md

```diff
@@ -0,0 +1,285 @@
+ones from the `Gateway` (up in the hierarchy). It is important to note that there is no way for the
+policy to discriminate the traffic it applies to at this stage. All traffic, as described above, would
+be affected by the merge of the two policies:
+
+```yaml
+enabled: true
+rps: 500
+```
+
+>**Note:** The status block of a policy should reflect the merged state of a policy as per other
+>policies at that level of the hierarchy.
+
+Any other `HTTPRoute` attached to the `my-gw` would still use the 42 rps limit. If we now delete the
+`FakeRLPolicy` attached to `my-gw`, the resulting policy would look like this:
+
+```yaml
+enabled: false
+rps: 500
+```
+
+Which would effectively disable all rate limiting traffic to the `foo-route` and ultimately the `foo`
+`Backend`.
+
+Using another example, let's consider a single `HTTPRoute` with two `HTTPRouteRule`s:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: bar-route
+spec:
+  parentRefs:
+  - name: example-gateway
+  hostnames:
+  - "bar.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /login
+    backendRefs:
+    - name: bar-svc
+      port: 8081
+  - matches:
+    - headers:
+      - type: Exact
+        name: env
+        value: production
+    backendRefs:
+    - name: bar-svc
+      port: 8080
+```
+
+It is important to consider that only _one_ rule will ever be match for any given requests. It makes for a simpler model to have only one "Policy" to also only ever being match, aligning the both behaviors. To apply a policy on the first rule only, the user needs to duplicate that match on the policy targeting `bar-route`:
+
+```yaml
+kind: FakeRLPolicy
+spec:
+  default:
+    rps: 50
+    rules:
+    - matches:
+      - path:
+          type: PathPrefix
+          value: /login
+  targetRef:
+    kind: HTTPRoute
+    name: foo-route
+```
+
+Hereby limiting the amount of hits to our `/login` on port `8081` to a 50 requests per second. That limit _will not_
+apply to other matches. So that Limitador will be configured by the Kuadrant controller to not rate limit traffic going
+to port `8080`.
+
+Merging only occurs at the `HTTPRoute` or within individual `HTTPRouteRule` within a route. The merging strategy will
+resolve the individual `HTTPRouteRule` all the policies boil down to and merge according to the Gateway API Policy
+Attachment hierarchy. Targeting a complete `HTTPRoute` is the equivalent of targeting each indidividual `HTTPRouteRule`s
+that composes the route.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+So there are a couple of things to consider here:
+
+- status reporting on "not yet" existing `HTTPRouteRule`, e.g. a Policy declares a set of `matches` that don't (yet?)
+  exist. The policy must _not_ be enforced... yet. But could eventually. This needs proper reporting. 
```

Comment:
An alternative could be report the policy as `invalid`, that could simplify things at least at the beginning... a typical error would be `RateLimitPolicy/AuthPolicy rules don't match the HTTPRouteRule definition`